pkg/api/tests.go

```diff
@@ -34,6 +34,40 @@ func PrintTestsDetailsJSONFromDB(w http.ResponseWriter, release string, testSubs
 	RespondWithJSON(http.StatusOK, w, responseStr)
 }
 
+func PrintTestOutputsFromDB(dbc *db.DB, release, test string, filters *filter.Filter, quantity int) ([]apitype.TestOutput, error) {
```

Comment:
Suggest GetTestOutputsFromDB or Load or something like that, we don't actually write to the http response here.